cmd/datamon/cmd/bundle_list.go

```diff
@@ -23,17 +25,18 @@ var BundleListCommand = &cobra.Command{
 		if err != nil {
 			logFatalln(err)
 		}
-		bundleDescriptors, err := core.ListBundles(params.repo.RepoName, remoteStores.meta, core.ConcurrentBundleList(params.core.ConcurrencyFactor))
-		if err != nil {
-			logFatalln(err)
-		}
-		for _, bd := range bundleDescriptors {
+		err = core.ListBundlesApply(params.repo.RepoName, remoteStores.meta, func(bundle model.BundleDescriptor) error {
 			var buf bytes.Buffer
-			err := listLineTemplate.Execute(&buf, bd)
-			if err != nil {
-				log.Println("executing template:", err)
+			e := listLineTemplate.Execute(&buf, bundle)
+			if e != nil {
+				log.Println("executing template:", e)
```

Comment:
This was not caught earlier but we should use fmt or change the logger to print to STDOUT. Right now this is printing to STDERR which makes it hard to script the output.

Reply:
sure. Will do.